index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" href="./style.css">
+      <title>Document</title>
+      <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+</head>
+
+<body>
+      <header>
+            <nav class="nav">
+                  <a>My logo</a>
+                  <img src="./image/Menu.png" alt="">
```

Comment:
- [ ]An `<a>` always has an URL, it would be better if you implement to follow HTML best practices.
```suggestion
            <nav class="nav">
                  <a href="#">My logo</a>
                  <img src="./image/Menu.png" alt="">
```